alws/routers/errata.py

```diff
@@ -152,6 +158,18 @@ async def release_errata_record(
     db_record.last_release_log = None
     await session.commit()
     release_errata.send(record_id, force)
+    if settings.github_integration_enabled:
```

Comment:
What if release fails? Shouldn't be more appropriate to have the dramatiq task updating the status?

Reply:
You right. I'll move this functionality where we set the status for Errata - "Released"

Reply:
Didn't you move this to dramatiq already?

Reply:
Yes, I did... I think something broke after rebase